webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
```

Comment:
Please decompose config to .common, .dev, and .prod

Reply:
Done.